install_scripts/blazegraph.sh

```diff
@@ -0,0 +1,20 @@
+echo "Installing Blazegraph"
+
+SHARED_DIR=$1
+
+if [ -f "$SHARED_DIR/install_scripts/config" ]; then
+  . $SHARED_DIR/install_scripts/config
+fi
+
+if [ ! -f "$DOWNLOAD_DIR/bigdata-bundled.jar" ]; then
+  echo -n "Downloading Blazegraph..."
+  wget --no-verbose -O "$DOWNLOAD_DIR/bigdata-bundled.jar" "http://iweb.dl.sourceforge.net/project/bigdata/bigdata/$BLAZEGRAPH_VERSION/bigdata-bundled.jar"
+  echo " done"
+fi
+
+cd $DOWNLOAD_DIR
+
+chown -hR vagrant:vagrant bigdata-bundled.jar
+
+echo "Starting Blazegraph"
+java -server -Xmx4g -jar bigdata-bundled.jar > /dev/null &
```

Comment:
You might run into problems with the -Xmx4g when the [Vagrantfile only has 2GB](https://github.com/ucsdlib/blazegraph-vagrant/pull/1/files#diff-23b6f443c01ea2efcb4f36eedfea9089R17).